updateDB.js

```diff
@@ -0,0 +1,88 @@
+var path = require('path')
+var fs = require('fs')
+var findUp = require('find-up')
+var execSync = require('child_process').execSync
+
+var BrowserslistError = require('./error')
+
+function updateDB () {
+  function getLastVersion () {
+    var output = execSync('npm show caniuse-lite version').toString()
+    var lastVersion = output.split('\n')[0]
+
+    return lastVersion
+  }
+
+  function startsWithTwoSpaces (line) {
+    return /^ {2}/.test(line)
+  }
+
+  var lockfile, packageManager
+
+  var packageJson = findUp.sync('package.json')
+  if (!packageJson) {
+    throw new BrowserslistError('Cannot find package.json')
```

Comment:
Bad error message. You do not explain what happened and how to fix it.

For instance, user can call Browserslist from wrong dir.